validator/validator_test.go

```diff
@@ -0,0 +1,45 @@
+package validator
+
+import (
+	"regexp"
+	"testing"
+)
+
+func TestProjectKey(t *testing.T) {
+	validProjectKeys := []string{
+		"abc", // min 3
+		"abcde12345", // max 10
+		"abc-123", // hyphen is supported but not documented
+	}
```

Comment:
Is `-abc123` a valid project key?

Reply:
No. I can add that to the negative tests.